Tiltfile

```diff
@@ -26,6 +26,13 @@ Here's a quick rundown of these services and their properties:
   * Other notes: Uses yarn. Does a `yarn install` for package dependencies, only if the dependencies have changed
 """
 
+
+# Read an option from the file tilt_option.json
```

Comment:
It might be worth providing a bit of "whether you should care" here, e.g.
"if you get push errors, rewrite image names to use a registry where you have push access"

Reply:
How's this?